packages/editor/src/components/StructureTree/StructureTree.tsx

```diff
@@ -49,10 +48,19 @@ export const StructureTree: React.FC<Props> = props => {
       // wait the component tree to be expanded
       setTimeout(() => {
         const selectedElement: HTMLElement | undefined | null =
-          wrapperRef.current?.querySelector(`#tree-item-${selectedId}`);
+          scrollWrapper.current?.querySelector(`#tree-item-${selectedId}`);
 
-        if (selectedElement) {
-          scrollIntoView(selectedElement, { time: 0, align: { lockX: true } });
+        const wrapperRect = scrollWrapper.current?.getBoundingClientRect();
```

Comment:
May cache the wrapper position information instead of re-reading it every time?

Reply:
I think it is not necessary. Because it only runs when being clicked. And the wrapper size will change if user resize it or browser. So there is no benefits to cache it and will bring more problems.

Reply:
OK.